both/api/sources/server/stream-chain/stream-types/transform-data/get-context.js

```diff
@@ -0,0 +1,35 @@
+import fs from 'fs';
+import vm from 'vm';
+import path from 'path';
+import getCategories from './get-categories';
+import getHelpersContent from './get-helpers-content';
+
+const LIBS_DIR_PATH = path.join(process.env.PWD, __dirname, 'libs');
```

Comment:
If I understand the API correctly, this will not let us build the list of scripts dynamically as it currently happens for [helpers](https://github.com/sozialhelden/accessibility-cloud/pull/21/commits/b9d870d3a55febe55f632cf5ff2c7e5359005749#diff-d747fcde5b37e2733ff5f02a35416695R18) and [libraries](https://github.com/sozialhelden/accessibility-cloud/pull/21/commits/b9d870d3a55febe55f632cf5ff2c7e5359005749#diff-6a1f17b861839fc0a03c00ed40ac91fdR11). In that case, I'll have to hard-code that list.